database/createMockData.js

```diff
@@ -30,48 +18,41 @@ const get_random = function (array) {
    lot: function (min,max) {
      return Math.round(Math.random() * (max-min) + min)
     },
-   pricePerSqft: "TBD"
- 
+   appliances: ['Dishwasher','Dryer', 'Garbage disposal', 'Refrigerator', 'Washer'],
+    intHeating: ['Forced air', 'Fan', 'Speeder air'],
+    intCooling: ['None', 'A/C', 'Central'],
+    kitchen_features: ['Counter', 'Pantry', 'Updated Kitchen', 'Eat In Kitchen'],
+    flooring: function (min,max) {
+     return Math.round(Math.random() * (max-min) + min)}
  };
- 
+
  const createFeatures = (callback) => {
    var features = [];
      for (let i = 0; i < 100; i++) {
        features.push({
-         Type: get_random(houseFeatures.type),
-         Year_Built: houseFeatures.year_built(1900,2006),
-         Heating: get_random(houseFeatures.heating),
-         Cooling: get_random(houseFeatures.cooling),
-         Parking: get_random(houseFeatures.parking),
-         Lot: houseFeatures.lot(1000,3000)+' sqft',
-         Price: "hello"
+         type: get_random(houseFeatures.type),
+         year_built: houseFeatures.year_built(1900,2006),
+         heating: get_random(houseFeatures.heating),
+         cooling: get_random(houseFeatures.cooling),
+         parking: get_random(houseFeatures.parking),
+         lot: houseFeatures.lot(1000,3000),
+         bedrooms: Math.round(Math.random() * 4),
+         bathrooms: Math.round(Math.random() * 4),
+         appliances: get_random(houseFeatures.appliances),
+         int_heating: get_random(houseFeatures.intHeating),
+         int_cooling: get_random(houseFeatures.intCooling),
+         kitchen:  get_random(houseFeatures.kitchen_features),
+         flooring: houseFeatures.flooring(1000,3000),
+         sqft: houseFeatures.flooring(1000,3000),
+         days_on_zillow: houseFeatures.flooring(1, 100)
        });
      }
    return features;
```

Comment:
Great job here! I just noticed some small spacing issues on the properties that do not use the "get_random" func. I would just separate the two param numbers by a space after each coma. Example: 

`line 45: flooring: houseFeatures.flooring(1000, 3000)`

Reply:
Gotcha.